server/src/main/resources/application.properties

```diff
@@ -1,2 +1,14 @@
+logging.level.com.coincoinche:DEBUG
 
-logging.level.com.coincoinche:DEBUG
\ No newline at end of file
+## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
+spring.datasource.url=jdbc:postgresql://localhost:5432/main
```

Comment:
Can we use an environment variable instead of hardcoding the URL here?

By the way, in the `docker-compose` file, you can easily add environments variables to a container and you should be able to connect to the database by using the container name as the host: `jdbc:postgresql://database:5432/main`. For local development, if we don't want to use Docker, we would just need to set the environment variable to `localhost` (but we need to figure out a way to have a proper `docker-compose` for development).